tests/modules/projects/test_models.py

```diff
@@ -110,8 +111,13 @@ def test_project_permission(db, regular_user):
     logging.info(temp_user.project_membership_enrollments)
     logging.info(temp_proj.user_membership_enrollments)
 
-    assert temp_user.has_permission_to_read(temp_encounter) is True
-    assert regular_user.has_permission_to_read(temp_encounter) is False
+    assert (
+        temp_user.can_perform_action(temp_encounter, ObjectAccessOperation.READ) is True
+    )
+    assert (
```

Comment:
likewise, we can use `assert not regular_user.can_perform_action(temp_encounter, ObjectAccessOperation.READ)`

Reply:
Agreed